packages/expo-blur/ios/TintStyle.swift

```diff
@@ -0,0 +1,72 @@
+import ExpoModulesCore
+
+public enum TintStyle: String, Enumerable {
+  case systemDefault = "default"
+  case extraLight
+  case light
+  case dark
+  case regular
+  case prominent
+  case systemUltraThinMaterial
+  case systemThinMaterial
+  case systemMaterial
+  case systemThickMaterial
+  case systemChromeMaterial
+  case systemUltraThinMaterialLight
+  case systemThinMaterialLight
+  case systemMaterialLight
+  case systemThickMaterialLight
+  case systemChromeMaterialLight
+  case systemUltraThinMaterialDark
+  case systemThinMaterialDark
+  case systemMaterialDark
+  case systemThickMaterialDark
+  case systemChromeMaterialDark
+
+  func toBlurEffect() -> UIBlurEffect.Style {
+    switch self {
+    case .systemDefault:
+      return .light
```

Comment:
Previously we used `light` for `default` and `extraLight` for `light`. Shouldn't `default` be `regular` now? 🤔
I would expect `default` to adapt to user's theme settings.

Reply:
Yeah, you're right, `default` should be `regular`. We can match the other's one to one now though right?

Reply:
Yes, sounds good 👍